chromium_src/chrome/browser/ui/views/accelerator_table.cc

```diff
@@ -19,10 +19,14 @@ const AcceleratorMapping kBraveAcceleratorMap[] = {
     // Command-Option-N
     {ui::VKEY_N, ui::EF_ALT_DOWN | ui::EF_PLATFORM_ACCELERATOR,
      IDC_NEW_OFFTHERECORD_WINDOW_TOR},
+    // Command-B
+    {ui::VKEY_B, ui::EF_PLATFORM_ACCELERATOR, IDC_TOGGLE_SIDEBAR},
```

Comment:
I don't think you need this `#ifdef` - `EF_PLATFORM_ACCELERTOR` is `Ctrl` on Win/Linux and `Cmd` on `OSX`

Reply:
Oh, you're right. Fixed! 👍🏼